tests/test_agent_equipment_preparation_authority.py

```
@@ -0,0 +1,1695 @@
+                {"$schema": "https://json-schema.org/draft/2020-12/schema"}
+            ),
+        }
+
+        class SplitSchemas(Mapping[str, bytes]):
+            def __init__(self) -> None:
+                self.reads = 0
+
+            def __len__(self) -> int:
+                return len(legitimate)
+
+            def __iter__(self):  # type: ignore[no-untyped-def]
```

Ruff (0.16.2)

[warning] Missing return type annotation for special method `__iter__`

(ANN204)